src/ImageSharp/Colors/ColorVector.Definitions.cs

```diff
@@ -0,0 +1,728 @@
+﻿// <copyright file="ColorVector.Definitions.cs" company="James Jackson-South">
```

Comment:
I wanted `Color` and `ColorVector` to have the same available static properties without having to directly call that file.